src/components/molecules/Wallet/Details.tsx

```diff
@@ -23,6 +25,33 @@ export default function Details(): ReactElement {
     setProviderInfo(providerInfo)
   }, [web3Provider])
 
+  console.log(config.oceanTokenAddress)
+
+  function addOceanToWallet() {
+    provider.sendAsync(
+      {
+        method: 'metamask_watchAsset',
+        params: {
+          type: 'ERC20',
+          options: {
+            address: config.oceanTokenAddress,
+            symbol: 'OCEAN',
+            decimals: 18,
+            image: Logo
```

Comment:
would be great if this works but not sure it will. Found the docs: https://docs.metamask.io/guide/registering-your-token.html#code-free-example

They say "A string url of the token logo" so I suppose they will always fetch it remotely. For which we have an asset in our `art` repo: https://github.com/oceanprotocol/art/blob/main/logo/token.png

So we should be able to use the raw URL for it: `https://raw.githubusercontent.com/oceanprotocol/art/main/logo/token.png`

Reply:
Right, I'll change that